libraries/libfc/include/fc/crypto/bls_signature.hpp

```diff
@@ -78,9 +78,9 @@ namespace fc::crypto::bls {
       }
 
       size_t get_hash() const;
-      static bls12_381::g2 to_jacobian_montgomery_le(const bls::signature_data::std_array_type& affine_non_montgomery_le);
+      static bls12_381::g2 to_jacobian_montgomery_le(const bls::signature_data& affine_non_montgomery_le);
    private:
-      bls::signature_data::std_array_type _affine_non_montgomery_le{};
+      bls::signature_data      _affine_non_montgomery_le{};
```

Comment:
should these be `const` as they are only set in constructor?

Reply:
They have to be non-const for reflection